Options.cs

```diff
@@ -54,6 +54,9 @@ class Options
             HelpText = "Subpath to replace files in (skip all other files).")]
         public string Subpath { get; set; }
 
+        [Option('z', "withoutgzip", Required = false, DefaultValue = false,
```

Comment:
I think it makes more sense to have the flag be opt-in and not opt-out.

I mean `gzip = true` makes more sense than `withoutgzip = false`